### PR TITLE
Refactor power chassis alert with new event format

### PR DESCRIPTION
**Pull request by sunnyqianzhang**
@iceiilin @pengz1 @anhou

### CODE REVIEW

File: lib/jobs/ipmi-job.js [hunk 1]
```diff
@@ -415,15 +415,23 @@ function ipmiJobFactory(
...
         var self = this;
+        var tmp = {};
+        tmp.type = 'polleralert';
+        tmp.action = 'powerState.updated';
+        tmp.typeId = data.workItemId;
+        tmp.nodeId = data.node;
+        tmp.severity = "information";
+        tmp.payload = {
+	    states: {
+                last: self.cachedPowerState[data.workItemId] ? 'ON' : 'OFF',
```
Comment (anhou) [inline lib/jobs/ipmi-job.js]: indent

File: lib/jobs/ipmi-job.js [hunk 2]
```diff
@@ -415,15 +415,23 @@ function ipmiJobFactory(
      */
      IpmiJob.prototype.powerStateAlerter = Promise.method(function(status, data) {
         var self = this;
+        var tmp = {};
+        tmp.type = 'polleralert';
+        tmp.action = 'powerState.updated';
```
Comment (anhou) [inline lib/jobs/ipmi-job.js]: it should be lower case

File: lib/jobs/ipmi-job.js [hunk 3]
```diff
@@ -415,15 +415,23 @@ function ipmiJobFactory(
...
+        tmp.typeId = data.workItemId;
+        tmp.nodeId = data.node;
+        tmp.severity = "information";
+        tmp.payload = {
+	    states: {
+                last: self.cachedPowerState[data.workItemId] ? 'ON' : 'OFF',
+		current: status.power ? 'ON' : 'OFF'
+	    },
+            nodeRef: '/nodes/' + data.node,
```
Comment (anhou) [inline lib/jobs/ipmi-job.js]: indent

**Comment by iceiilin** [comment]
:+1:

**Pull request closed without merging**